app/controllers/authentication/sessions_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Authentication::SessionsController < ApplicationController
```

Comment:
📝

This is a sort of jump controller that decides where to take the user. Adjacently, the `authenticated` route from devise in `routes.rb` has been removed and this general root handler is now what decides.